src/pages/CommitDetailPage/Header/HeaderTeam/HeaderTeam.jsx

```diff
@@ -0,0 +1,103 @@
+import { useParams } from 'react-router-dom'
+
+import { getProviderCommitURL, getProviderPullURL } from 'shared/utils'
+import { formatTimeToNow } from 'shared/utils/dates'
+import A from 'ui/A'
+import CIStatusLabel from 'ui/CIStatus'
+import Icon from 'ui/Icon'
+import TotalsNumber from 'ui/TotalsNumber'
+import TruncatedMessage from 'ui/TruncatedMessage/TruncatedMessage'
+
+import { useCommitHeaderDataTeam } from './hooks'
+
+import PullLabel from '../PullLabel'
+
+function HeaderDefault() {
```

Comment:
Quick nitpick, can you update this to reflect that this is the team header?

Reply:
ahh I missed this in the copy pasting, yeap will do!

Reply:
Thaaank you, thank you